cubi_tk/snappy/pull_data_common.py

```diff
@@ -31,7 +31,7 @@ def filter_irods_collection(self, identifiers, remote_files_dict, file_type):
         :type identifiers: list
 
         :param remote_files_dict: Dictionary with iRODS collection information. Key: file name as string (e.g.,
-        'P001-N1-DNA1-WES1.vcf.gz'); Value: iRODS data (``IrodsDataObject``).
+        'P001-N1-DNA1-WES1.vcf.gz'); Value: iRODS data (``DataObject``).
```

Comment:
I think the correct calls/type name here is `iRODSDataObject` (at least its imported under that name in check.py; inside irods it iirc called `data_object`)